lib/backends/cpu/op-resolve-rules.ts

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import {FLOAT_TYPES, NUMBER_TYPES} from '../../operators';
+import {OpSet} from '../../opset';
+
+import {CpuArgMax} from './ops/argMax';
+import {CpuBatchNormalization} from './ops/batch-normalization';
+import {CpuBinaryOp} from './ops/binary-op';
+import {CpuConcat} from './ops/concat';
+import {CpuConv} from './ops/conv';
+import {CpuDropout} from './ops/dropout';
+import {CpuFlatten} from './ops/flatten';
+import {CpuGather} from './ops/gather';
+import {CpuGemm} from './ops/gemm';
+import {CpuImageScaler} from './ops/image-scaler';
+import {CpuInstanceNormalization} from './ops/instance-normalization';
+import {CpuLrn} from './ops/lrn';
+import {CpuMatMul} from './ops/matmul';
+import {CpuAveragePool, CpuGlobalAveragePool, CpuGlobalMaxPool, CpuMaxPool} from './ops/pool';
+import * as cpuReduce from './ops/reduce';
+import {CpuReshape} from './ops/reshape';
+import {CpuSlice} from './ops/slice';
+import {CpuSoftmax} from './ops/softmax';
+import {CpuSqueeze} from './ops/squeeze';
+import {CpuSum} from './ops/sum';
+import {CpuTile} from './ops/tile';
+import {CpuTranspose} from './ops/transpose';
+import * as unaryOps from './ops/unary-op';
+import {CpuUnsqueeze} from './ops/unsqueeze';
+
+export const CPU_OP_RESOLVE_RULES: ReadonlyArray<OpSet.ResolveRule> = [
+  ['Abs', '', '6+', () => new unaryOps.CpuUnaryOp(NUMBER_TYPES, unaryOps.abs)],
+  ['Acos', '', '7+', () => new unaryOps.CpuUnaryOp(NUMBER_TYPES, unaryOps.acos)],
+  ['Add', '', '7+', () => new CpuBinaryOp(NUMBER_TYPES, (e1, e2) => (e1 + e2))],
+  ['And', '', '7+', () => new CpuBinaryOp(['bool'], (e1, e2) => (e1 && e2))],
+  ['ArgMax', '', '1+', () => new CpuArgMax()],
+  ['Asin', '', '7+', () => new unaryOps.CpuUnaryOp(NUMBER_TYPES, unaryOps.asin)],
+  ['Atan', '', '7+', () => new unaryOps.CpuUnaryOp(NUMBER_TYPES, unaryOps.atan)],
+  ['AveragePool', '', '7+', () => new CpuAveragePool()],
+  ['BatchNormalization', '', '7+', () => new CpuBatchNormalization()],
+  ['Ceil', '', '6+', () => new unaryOps.CpuUnaryOp(NUMBER_TYPES, unaryOps.ceil)],
```

Comment:
Also, how can we ensure potentially a bad commit that could have conflicting operator resolves possible ?

For example, Ceil 6+ could be mapped to an op, let's say Ceil 10 has has a different op implementation, but the developer forgets to change `6+` to `6-9`. In that case, multiple op resolves are possible.

One way is to run a script at check-in time but we will have to maintain that...